### ISSUE TITLE
Support doxygen-style math delimiters

### ISSUE THREAD

**Issue by bilderbuchi** [goessner/mdmath#121]
### Summary

The Readme tells us that mdmath uses "[markdown-it-texmath's](https://github.com/goessner/markdown-it-texmath) support of different formula delimiters". 
In the list of Mdmath: Delimiters configuration settings, `doxygen` is conspicuously absent although it's listed in markdown-it-texmath's [Readme](https://github.com/goessner/markdown-it-texmath#features).

Using this markdown file
```md
Dollars:

$$
  L[y] := y'' - V y = \rho(x).
$$

Brackets:

\[
  L[y] := y'' - V y = \rho(x).
\]

Doxygen:

\f[
  L[y] := y'' - V y = \rho(x).
\f]
```
I tested different settings for the Mdmath: Delimiters settings. 

`brackets` works correctly. Interestingly I need a VS Code _restart_ for the setting to become active.

### Expected behavior
Using the `doxygen`setting, I expect the formula to be rendered.

### Actual behavior
Unfortunately, using `doxygen` does not seem to do anything - the code remains raw and not rendered into a formula
![image](https://user-images.githubusercontent.com/327442/155490504-445c458c-545b-471a-bd86-3442cf93c764.png)
(I suspect that the dollars variant keeps getting rendered because of base markdown extension support)

### Environment

```text
     Operating system: windows 10
       VSCode version: 1.64.2
Markdown+Math version: 2.7.4
```

**Comment by bilderbuchi** [comment]
@goessner is there anything I can do on my side (tweak settings in a file or so) to enable doxygen math delimiters in mdmath, Stefan?

**Comment by goessner** [comment]
Yes ... please try to add doxygen delimiters to the delimiters array in

https://github.com/goessner/markdown-it-texmath

I will come back to markdown-it-texmath and mdmath in two weeks at the earliest.

thanks

**Comment by bilderbuchi** [comment]
Thank you! I tried to poke around and do something like this already, but unfortunately I could not locate the files related to the VS Code extension on my disk. (I'm not a web/JS person, so that will also not help)
Doxygen delimiters are already present, though, as far as I can tell: https://github.com/goessner/markdown-it-texmath/blob/b08ddd342f26ddcdc3afa2218f1b55410f44029b/texmath.js#L215-L234 

The point I tried to make in the issue description is that support seems to be there in markdown-it-texmath, but it does _not_ work in mdmath.

**Comment by goessner** [comment]
Ahh ... I see. In fact I need to update mdmath to current markdown-it-texmath version. This will take same time beside other actions.

What you can try to do is replace in your folder

`/Users/<name>/.vscode/extensions/mdmath/node-modules/markdown-it-texmath`

the `texmath.js` file or the complete folder. If it works, it is fine, if not, you are lost and have to wait for my next update.

Alternatively you can copy the doxygen stuff into your `texmath.js` file.

**Comment by bilderbuchi** [comment]
Aahh, I was looking in APPDATA...
Yes, replacement of the folder seems to have worked! I get preview of equations now. 

Curiously, it seems empty lines are neecessary before starting an equation block (for it to be rendered correctly), which I'm not sure is expected for markdown, but I guess that's a markdown-it-texmath limitation?

Also, the [`\f{environment}{`](https://www.doxygen.nl/manual/commands.html#cmdfcurlyopen) Doxygen formula syntax does not work, but I think getting to that is probably more effort. I'll use manual enviroments `\begin{eqnarray}`/`end{eqnarray}` inside the block as a workaround.
